packages/esm-generic-patient-widgets-app/src/config-schema.ts

```diff
@@ -23,6 +23,9 @@ export const configSchema = {
       color: {
         _type: Type.String,
       },
+      decimalPlaces: {
+        _type: Type.Number,
```

Comment:
It would be nice to have a validator here, e.g.:

```ts
_validator: [validators.inRange(0, 10)]
```

(`validators` above should be importable from `@openmrs/esm-framework`).